packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -3031,6 +3031,11 @@ export type OmnichannelEndpoints = {
 		PUT: (params: PUTLivechatPriority) => void;
 	};
 
+	'/v1/livechat/priority.reset': {
```

Comment:
```suggestion
	'/v1/livechat/priorities.reset': {
```

Pls also change on the other references 🙏